tests/catalog/test_hive.py

```diff
@@ -1195,3 +1195,40 @@ def test_hive_wait_for_lock() -> None:
     with pytest.raises(WaitingForLockException):
         catalog._wait_for_lock("db", "tbl", lockid, catalog._client)
     assert catalog._client.check_lock.call_count == 5
+
+def test_create_hive_client_success():
+    properties = {
+        "uri": "thrift://localhost:10000",
+        "ugi": "user"
+    }
+
+    with patch('pyiceberg.catalog.hive._HiveClient', return_value=MagicMock()) as mock_hive_client:
+        client = HiveCatalog._create_hive_client(properties)
+        mock_hive_client.assert_called_once_with("thrift://localhost:10000", "user")
+        assert client is not None
+
+def test_create_hive_client_multiple_uris():
+    properties = {
+        "uri": "thrift://localhost:10000,thrift://localhost:10001",
+        "ugi": "user"
+    }
+
+    with patch('pyiceberg.catalog.hive._HiveClient') as mock_hive_client:
+        mock_hive_client.side_effect = [Exception("Connection failed"), MagicMock()]
```

Comment:
does this mean the first connection will fail?

Reply:
Thank you for your comments. Yes this means the first connections will fail.

Scenario `2 uris; both passes` will never be a valid case, if the first uri passes the second is never tried. So the only additional test would be to add one covering `(optional) 2 uris; 1 pass, second one not tried`

Reply:
thanks! i think that should cover all the cases